### ISSUE TITLE
[cpackget] Installing a local pack that does not exist triggers error message twice

### ISSUE THREAD

**Issue by ReinhardKeil** [Open-CMSIS-Pack/cpackget#151]
**Describe the bug**
Error message appears twice.

**To Reproduce**

Run the cpackget with the following args:
```
C:\w> cpackget add .\STM32U575I-EV_BSP\Keil.STM32U575I-EV_BSPx.pdsc
I: Using pack root: "C:\Users\reikei01\AppData\Local\Arm\Packs" (default mode - no specific CMSIS_PACK_ROOT chosen)
I: Adding pdsc ".\STM32U575I-EV_BSP\Keil.STM32U575I-EV_BSPx.pdsc"
E: open .\STM32U575I-EV_BSP\Keil.STM32U575I-EV_BSPx.pdsc: The system cannot find the file specified.
E: open .\STM32U575I-EV_BSP\Keil.STM32U575I-EV_BSPx.pdsc: The system cannot find the file specified.
```

**Expected behavior**

The error message should appear just once
`E: open .\STM32U575I-EV_BSP\Keil.STM32U575I-EV_BSPx.pdsc: The system cannot find the file specified.`

**Environment (please complete the following information):**
 - OS: Windows
 - Version 10

**Additional context**

**Comment by jkrech** [comment]
@lud0v1c could you please comment, whether there is a reason for printing this message twice? Are you looking for the pdsc file in two different locations. E.g. whether it is a public pack and the pdsc is listed in index.pidx or whether it is a local pack which has the pdsc in the .Local directory. Without the additional information the duplicated error message is not meaningful to the user.

**Comment by thorstendb-ARM** [comment]
Fixed by checking against the prev. printed message. Did not find a way to extend the error "class" as it is also used by external libraries.

**Comment by jkrech** [comment]
fixed in cpackget starting version 0.9.4
